app/controllers/Checkout.scala

```diff
@@ -268,6 +270,20 @@ object Checkout extends Controller with LazyLogging with CatalogProvider {
         _ + _
       }
 
+      val acquisitionData = request.session.get("acquisitionData")
+      if (acquisitionData.isEmpty) {
+        logger.warn(s"No acquisitionData in session")
+      }
+
+      val promotion = subscribeRequest.genericData.promoCode.map(_.get).flatMap(code => tpBackend.promoService.findPromotion(NormalisedPromoCode.safeFromString(code)))
+
+      OphanService(isTestService = tpBackend.environmentName != "PROD")
+        .submit(SubscriptionAcquisitionComponents(subscribeRequest, promotion, acquisitionData))
+        .bimap(
+          err => logger.warn(s"Error submitting acquisition data to Ophan. ${err}"),
+          submission => logger.info(s"Submitted acquisition data to Ophan. ${submission}")
```

Comment:
Just checking, are we ok logging the submission considering it contains e.g. the browser id ?

Reply:
Good point, this is more for debug purposes - will remove